prestodb/cdh5-base/Dockerfile

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM teradatalabs/centos6-java8-oracle:unlabelled
-MAINTAINER Teradata Docker Team <docker@teradata.com>
+FROM prestodb/centos6-oj8:unlabelled
+MAINTAINER Presto community <https://prestodb.io/community.html>
```

Comment:
the best would be to do a single change per commit. Here is plenty of changes... which makes it hard to review.

Reply:
sorry, will behave better next time